src/main.rs

```diff
@@ -52,6 +62,22 @@ fn main() {
         }),
         ..Default::default()
     };
+
+    let speed_mode: usize = matches
+        .value_of("SPEED_MODE")
+        .map(|val| val.parse().expect("Speed mode must be in range [0; 2]"))
+        .unwrap();
+
+    if matches.occurrences_of("SCENE_CHANGE_DETECTION_SPEED") != 0 {
```

Comment:
You can use `matches.is_present` instead of `occurrences_of`, it avoids the need to compare against 0. (Same with line 90 below.)